presto-main/src/main/java/com/facebook/presto/tracing/OpenTelemetryBuilder.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tracing;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+public final class OpenTelemetryBuilder
+{
+    private OpenTelemetryBuilder()
+    {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated.");
+    }
+
+    public static OpenTelemetry build(boolean tracing)
+    {
+        Resource resource = Resource.getDefault()
+                .merge(Resource.create(Attributes.of(ResourceAttributes.SERVICE_NAME, "presto")));
+
+        SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+                .setResource(resource)
+                .build();
+
+//                .addSpanProcessor(BatchSpanProcessor.builder(OtlpGrpcSpanExporter.builder().build()).build())
+
+        OpenTelemetry openTelemetry;
+        if (tracing) {
+            openTelemetry = OpenTelemetrySdk.builder()
+                    .setTracerProvider(sdkTracerProvider)
```

Comment:
had it there initially because of how i structured the tracer provider, but changed that. so there's no case to initialize it without tracing. i can remove it.

Reply:
sounds good

Reply:
sounds good, lets simplify this